Sources/SKSwiftPMWorkspace/SwiftPMBuildSystem.swift

```diff
@@ -544,7 +544,7 @@ extension SwiftPMBuildSystem: SKCore.BuildSystem {
       return targets.map(ConfiguredTarget.init)
     }
 
-    if path.basename == "Package.swift" {
+    if path.basename == "Package.swift" && projectRoot.asURL.equalResolvingSymlinksInPath(path.parentDirectory.asURL) {
```

Comment:
`projectRoot` already has symlinks resolved, so you only need to resolve symlinks for `path.parentDirectory`.

```suggestion
    if path.basename == "Package.swift" && projectRoot == (try? TSCBasic.resolveSymlinks(TSCBasic.AbsolutePath(path.parentDirectory))) {
```